### ISSUE TITLE
No File Name at Standard Output

### ISSUE THREAD

**Issue by alvivi** [stil4m/elm-analyse#129]
Now there is no way to know which is the file causing an error. At least using the cli and the `human` output...

Now (0.13.1)
```
Messages:
Duplicate import for module `E`` at [ ((5,0),(5,41)) | ((6,0),(6,20)) ]
Unused variable `b` inside pattern at ((35,26),(35,34))
```

Before  (0.12.1):
```
Messages:
Duplicate import for module `E`in file "src/a.elm" at [ ((5,0),(5,41)) | ((6,0),(6,20)) ]
Unused variable `b` inside pattern in file "src/b.elm" at ((35,26),(35,34))
e
```

**Comment by stil4m** [comment]
Good point. I missed that. Will bring it back in a minor release.

**Comment by stil4m** [comment]
New output will be something like this:

```
Messages:
- src/Analyser/SourceLoadingStage.elm
  > Unused imported variable `empty` at ((5,26),(5,31))
  > Unused top level definition `foo` at ((24,0),(24,3))
- src/Analyser/State.elm
  > Unused imported variable `dict` at ((26,44),(26,48))
```

**Comment by stil4m** [comment]
Fixed in 0.13.2
